environment.yml

```diff
@@ -57,5 +56,7 @@ dependencies:
   - simplejson >=3.8.1
   - tornado >=5.0.0,<6.0.0
   - typing >=3.6.6
+  # Pin OpenSSL to avoid: https://github.com/DIRACGrid/DIRAC/issues/4489
+  - openssl <1.1
```

Comment:
https://github.com/DIRACGrid/DIRAC/issues/4489

Reply:
Related to https://github.com/DIRACGrid/DIRAC/pull/4391

Reply:
Not clear... but may be... I'll add a comment to the PR to link the two